File: MODUL-4/python/PRAK403-2310817210007-RaymondHariyono.py
batas1,batas2 = (input().split())
batas1 = int(batas1)
batas2 = int(batas2)
if batas1 > batas2:
    for i in range(batas2, batas1+1):
        print(i, batas1 + batas2 - i, end=" - ")
else:
    for i in range(batas2, batas1-1, -1):
        print(i, batas1 + batas2 - i, end=" - ")

File: MODUL-4/C/PRAK403-2310817210007-RaymondHariyono.c
#include <stdio.h>

int main (){
int batas1;
int batas2;
scanf("%i%i",&batas1,&batas2);

    if(batas1 > batas2){
        for(int i= batas2; i <= batas1; i++){
        printf("%d %d - ", i, batas1 + batas2 - i);
    }
}
    else {
        for(int i= batas2; i >= batas1; i--){
        printf("%d %d - ", i, batas1 + batas2 - i);}
}
return 0;
}